float64[7] commanded
float64[7] measured
float64[7] error
float64 root_mean_square_error
